onboarding/style.css

```diff
@@ -56,6 +58,7 @@ button:hover {
     position: relative;
     top: 0px;
     transition: opacity .3s, border .3s, top .3s;
+    color: black;
```

Comment:
I don’t think that the button’s color should change when it’s hovered, the position of the button already does.

Reply:
oh, mb didnt see that